### PR TITLE
[WIP] Poretools

### PR DESCRIPTION
**Pull request by shiltemann**
[Poretools:](http://poretools.readthedocs.io) A toolkit for analyzing Nanopore sequence data

Flexible toolkit for exploring datasets generated by nanopore sequencing devices from MinION for the purposes of quality control and downstream analysis.

..more tools may follow

Please sign off by admins only
------------------------------

* [ ] Has no duplicate (otherwise merge required)
* [ ] Indentation is correct
* [ ] Tests
  - [ ] Parameters are reasonably covered
  - [ ] Test files are appropriate
* [ ] `<command/>`: file names are escaped
* [ ] Tool requires version/build increase
* [ ] Toolshed user `erasmus-medical-center` has access to associated toolshed repo(s)

### CODE REVIEW

File: tools/poretools/fast5_to_fastx.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<tool id="poretools_fastx" name="Fast5-to-FastX" version="@TOOL_VERSION@-g0.3">
```
Comment (yhoogstrate) [inline tools/poretools/fast5_to_fastx.xml:1]: What's with the version?

**Review by yhoogstrate — APPROVED** [review]

File: tools/poretools/fast5_to_fastx.xml (hunk 1 above)
Comment (shiltemann) [reply tools/poretools/fast5_to_fastx.xml:1]: oja, omdat ik m moest updaten om push naar testtoolshed goed te laten gaan, maar kan m wel weer op 0 zetten ja

File: tools/poretools/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: poretools
+owner: erasmus-medical-center
+categories:
+- Sequence Analysis
+description: |
+  A toolkit for analyzing Nanopore sequence data
+long_description: |
+  Flexible toolkit for exploring datasets generated by nanopore sequencing devices from MinION for the purposes of quality control and downstream analysis.
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/galaxytools-emc/tree/master/tools/poretools
```
Comment (yhoogstrate) [inline tools/poretools/.shed.yml:9]: homepage url: https://github.com/arq5x/poretools

File: tools/poretools/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: poretools
```
Comment (yhoogstrate) [inline tools/poretools/.shed.yml:1]: @shiltemann Gebruik je nanpore data voor een EMC project? Zoniet, is dit niet meer op zijn plaats in IUC?

**Comment by shiltemann** [comment]
superceded by IUC tool

**Pull request closed without merging**